source/blog/2010-05-30-css-id-and-class-naming.html.markdown

```diff
@@ -10,7 +10,7 @@ tags:
 
 <p>I&#39;m a developer, I like structure, I like clean code, I like DRY code. I find writing CSS incredibly painful.</p>
 <p>CSS1 was first published in December 1996, so I ask, why in 2010 does it feel like CSS is just as painful as it was 14 years ago?</p>
-<p><a href="http://www.flickr.com/photos/popandshorty/"><img alt="" src="http://farm3.static.flickr.com/2461/3970138375_3e68095deb_o.jpg" width="500" /></a></p>
```

Comment:
This photo isn't ours to have moved over, it's fine for this to still be a link to flickr.